classes/cookie.php

```diff
@@ -93,11 +95,27 @@ public static function set($name, $value, $expiration = null, $path = null, $dom
 		is_null($domain) and $domain = static::$config['domain'];
 		is_null($secure) and $secure = static::$config['secure'];
 		is_null($http_only) and $http_only = static::$config['http_only'];
+		is_null($same_site) and $same_site = static::$config['same_site'];
 
 		// add the current time so we have an offset
 		$expiration = $expiration > 0 ? $expiration + time() : 0;
+		
+		if (is_null($same_site)) {
+		    return setcookie($name, $value, $expiration, $path, $domain, $secure, $http_only);
+		}
+		
+		if (PHP_VERSION_ID < 70300) {
+		    return setcookie($name, $value, $expiration, "{$path}; samesite={$same_site}", $domain, $secure, $http_only);
+		}
 
-		return setcookie($name, $value, $expiration, $path, $domain, $secure, $http_only);
+		return setcookie($name, $value, [
```

Comment:
Fuel coding standards: don't use short array notation for PHP 5.x compatibility.

Reply:
fixed!